### PR TITLE
Updates for shipping methods B2B config ref and synch with procedures

### PR DESCRIPTION
**Pull request by leslietilling**
## Purpose of this pull request

This pull request (PR) includes additional user guide updates for the B2B shipping methods set in the config and for the company account. Screens are updated against latest build.

**Internal staging build at: 544/user-guide/v2.3/**

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable: 2.4.1

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

- https://docs-beta.magento.com/user-guide/configuration/general/b2b-features.html
- https://docs-beta.magento.com/user-guide/stores/b2b-features.html
- https://docs-beta.magento.com/user-guide/customers/account-company-create.html

## Links to Magento updates

https://github.com/magento/partners-magento2b2b/issues/160
https://github.com/magento/partners-magento2b2b/issues/162

whatsnew
Added a new section in the [B2B Features configuration reference](https://docs.magento.com/user-guide/configuration/general/b2b-features.html) for the Default B2B Shipping Methods. 

Updates to [Creating a Company Account](https://docs.magento.com/user-guide/customers/account-company-create.html) for shipping methods configured for the account from the Admin.

### CODE REVIEW

File: src/stores/b2b-features.md [hunk 1]
```diff
@@ -41,31 +41,41 @@ By default, all B2B features are initially disabled. However, they are always av
...
       [_B2B Configuration_]({% link configuration/general/b2b-features.md %})
 
-1. To establish a default payment method for B2B orders, set **Applicable Payment Methods** to one of the following:
+1. To establish the default payment method(s) for B2B orders, set **Applicable Payment Methods** to one of the following:
 
    - All Payment Methods
 
    - Specific Payment Methods
 
       For Specific Methods, select the **Payment Methods** that you want to make available to your customers by holding down the Ctrl key (PC) or the Command key (Mac) as you click each option.
```
Comment (hickskenh) [inline src/stores/b2b-features.md:50]: if specific methods is a field name, it should be bolded.  if not, should it be capped?

File: src/stores/b2b-features.md [hunk 2]
```diff
@@ -41,31 +41,41 @@ By default, all B2B features are initially disabled. However, they are always av
...
-    [_Default B2B Payment Methods_]({% link configuration/sales/payment-methods.md %})
+    _Default B2B Payment Methods_
 
-1. Configure shipping methods for B2B orders.
+1. To establish the default shipping method(s) for B2B orders, set **Applicable Shipping Methods** to one of the following:
 
    - All Shipping Methods
    - Specific Shipping Methods
 
      For Specific Shipping Methods, select the **Shipping Methods** that you want to make available to your customers by holding down the Ctrl key (PC) or the Command key (Mac) as you click each option.
```
Comment (hickskenh) [inline src/stores/b2b-features.md:67]: specific methods, same note

**Review by hickskenh — APPROVED** [review]
approved after your review.

File: src/stores/b2b-features.md (hunk 1 above)
Comment (leslietilling) [reply src/stores/b2b-features.md:50]: I am think not capped since that is not the title/label.

File: src/stores/b2b-features.md (hunk 2 above)
Comment (leslietilling) [reply src/stores/b2b-features.md]: done

**Comment by leslietilling** [comment]
running tests

**Pull request merged**